pkg/storaged/fsys-panel.jsx

```diff
@@ -124,6 +124,9 @@ export class FilesystemsPanel extends React.Component {
                     total += Number(fs.Used[1]);
             });
 
+            const overhead = pool.TotalPhysicalUsed[0] ? (Number(pool.TotalPhysicalUsed[1]) - total) : 0;
+            const pool_total = Number(pool.TotalPhysicalSize) - overhead;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18831-20230522-125636-1ec6e298-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)